### PR TITLE
Update to quantiseqr

### PR DESCRIPTION
**Pull request by czackl**
Changed to quantiseqr

### CODE REVIEW

File: NAMESPACE [hunk 1]
```diff
@@ -42,11 +41,9 @@ export(timer_available_cancers)
 export(xCell.data)
 import(ComICS)
 import(dplyr)
-import(limSolve)
 import(mMCPcounter)
 import(magrittr)
 import(methods)
-import(preprocessCore)
```
Comment (czackl) [inline NAMESPACE:49]: This is probably why the ABIS test failed, but should still be in here. Maybe need to Document again

**Comment by czackl** [comment]
Okay, tests look good except the conda run. I just put `quantiseqr` in the DESCRIPTION, still did not find it in the conda workflow. Any recommendations @LorenzoMerotto @federicomarini

**Comment by federicomarini** [comment]
Well, `quantiseqr` is also "regularly there" in Bioc, so bioconda should generate regularly the binaries.
Do these get picked up? https://anaconda.org/bioconda/bioconductor-quantiseqr

File: NAMESPACE (hunk 1 above)
Comment (czackl) [inline NAMESPACE:49]: I have no idea why preprocessCore and limSolve have been removed from the NAMESPACE, i just documented again...

**Comment by czackl** [comment]
> Well, `quantiseqr` is also "regularly there" in Bioc, so bioconda should generate regularly the binaries. Do these get picked up? https://anaconda.org/bioconda/bioconductor-quantiseqr

So should i put bioconductor-quantiseqr in the description instead of just quanitseqr? 🤔

**Comment by federicomarini** [comment]
I don't know how the GHA in detail is picking up the dependency, but the correct way "in the DESCRIPTION" is to simply say

```
Imports:
    ...
    quantiseqr
```

**Comment by LorenzoMerotto** [comment]
@czackl 
You need to add `bioconductor-quantiseqr` both in the `host` and the `run` sections here

https://github.com/omnideconv/immunedeconv/blob/41b78f76c7a94cba889388dfba18eb7d018142bd/.conda/meta.yaml#L18

Sorry if I haven't suggested the change directly but apparently I don't know how to do it on an unmodified file

**Comment by LorenzoMerotto** [comment]
To me it looks good. 
@czackl if you don't have anything else to add I'll merge

**Comment by czackl** [comment]
Perfect, let's merge. Thanks for the help!

**Comment by czackl** [comment]
@LorenzoMerotto all checks passed now

**Pull request merged**